scan.py

```diff
@@ -34,7 +34,7 @@
 
 
 def scan(rootdir):
-  req_files = list(set(glob(path.join(rootdir, "**/*requirements*.txt"))) | set(glob(path.join(rootdir, "*requirements*.txt"))))
+  req_files = list(set(glob(path.join(rootdir, "**/**/*require*.txt"))) | set(glob(path.join(rootdir, "**/*require*.txt"))) | set(glob(path.join(rootdir, "*require*.txt"))))
```

Comment:
this looks good, but we should add a comment at the very least describing this intended behavior